kubemarine/plugins/calico.py

```diff
@@ -24,19 +24,6 @@
 
 
 def enrich_inventory(inventory, cluster):
```

Comment:
With changes from [PR-381](https://github.com/Netcracker/KubeMarine/pull/381) this method will be empty.
Maybe remove it at all?

Reply:
Thank you for the suggestion. The method is removed.